src/detection/terminalfont/terminalfont.c

```diff
@@ -320,6 +320,28 @@ FF_MAYBE_UNUSED static bool detectKitty(const FFinstance* instance, FFTerminalFo
     return true;
 }
 
+static bool detectTerminator(const FFinstance* instance, FFTerminalFontResult* result)
+{
+    FFstrbuf fontName;
+    ffStrbufInit(&fontName);
+
+    FFpropquery fontQuery[] = {
+        {"font =", &fontName},
+    };
+
+    if(!ffParsePropFileConfigValues(instance, "terminator/config", 1, fontQuery))
+        return false;
```

Comment:
this can be simplified to:
```c
if(!ffParsePropFileConfig(instance, "terminator/config", "font =" &fontName))
    return false;
```

Reply:
Alright.... like i said, long time ago i wrote some C code :wink: i've read again about uint32_t typing - now i understand why i can do this if i have only one query :wink: thanks for the tip(s)